features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_the_same_tax_rate.feature

```diff
@@ -0,0 +1,50 @@
+@applying_taxes
+Feature: Applying correct taxes for item units with the same tax rate
+    In order to pay proper amount when buying goods from the same tax category
+    As a Visitor
+    I want to have correct taxes applied to my order
```

Comment:
As above 👆🏼

Reply:
I'm resolving these comments, as they litter the PR a bit, the main comment is enough